extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -95,6 +99,80 @@
   border-bottom-right-radius: 50% !important;
 }
 
+.i-amphtml-story-interactive-slider-average-bubble {
+  position: absolute !important;
+  transform: translate(-50%, -50%) scale(0) !important;
+  width: 2.8em !important;
+  height: 2.8em !important;
+  top: 50% !important;
+  left: var(--average) !important;
+  text-align: center !important;
+  line-height: 1em !important;
+  font-weight: 300 !important;
+  color: black !important;
+  z-index: 1 !important;
+}
+
+.i-amphtml-story-interactive-slider-average-bubble::after {
+  position: absolute !important;
+  content: '' !important;
+  box-sizing: border-box !important;
+  height: 100% !important;
+  width: 100% !important;
+  top: 0% !important;
+  background: black !important;
+  z-index: -1 !important;
+  left: 50% !important;
+  transform: translateX(-50%) rotate(45deg) !important;
+  border-radius: 50% 50% 0% 50% !important;
+  box-shadow: 0em 0em 0.5em rgba(0, 0, 0, 0.25) !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-average-bubble {
+  transform: translate(-50%, -50%) scale(0.5) !important;
+  left: var(--average) !important;
+  position: absolute !important;
+  top: 50% !important;
```

Comment:
These rules shouldn't be necessary, on `post-selection` you should only need to update the scale